analyzers/src/SonarAnalyzer.VisualBasic/Facade/VisualBasicSyntaxFacade.cs

```diff
@@ -83,10 +83,11 @@ node switch
         public override SyntaxNode NodeExpression(SyntaxNode node) =>
             node switch
             {
-                ArgumentSyntax argument => argument.GetExpression(),
-                InvocationExpressionSyntax invocation => invocation.Expression,
-                SyncLockStatementSyntax syncLock => syncLock.Expression,
-                InterpolationSyntax interpolation => interpolation.Expression,
+                ArgumentSyntax x => x.GetExpression(),
+                InterpolationSyntax x => x.Expression,
+                InvocationExpressionSyntax x => x.Expression,
+                SyncLockStatementSyntax x => x.Expression,
+                ReturnStatementSyntax x => x.Expression,
```

Comment:
CSharp has `ParenthesizedExpressionSyntax x => x.Expression,`, should it be here too? With an UT